kolibri/plugins/device_management/assets/src/views/manage-content-page/index.vue

```diff
@@ -140,6 +142,17 @@
             unblockCb();
           });
       },
+      updateChannelList: function() {
+        // Refresh channel list every 10 seconds.
+        var firstChannel = this.pageState.channelList[0];
+        if (this.channelRefreshCount < 10) {
+          if (!firstChannel.available) {
+            this.refreshChannelList();
+            this.channelRefreshCount += 1;
+            setTimeout(() => this.updateChannelList(), 10000);
```

Comment:
If you must use a setTimeout function, you need to make sure to clear the timeout in the component's `beforeDestroy` handler. Search for `clearTimeout` in .vue files for examples